src/amb_kf.c

```diff
@@ -524,6 +524,10 @@ void set_nkf_matrices(nkf_t *kf, double phase_var, double code_var,
                   kf->decor_obs_mtx);
 }
 
+/* Currently this function is only ever used to find the new position of a prn
+ * in a new basis (permuted list) of prns, so all callsites assert the result
+ * is not -1.
+ */
```

Comment:
Turn this into a Doxygen style comment with description of parameters and return values. There is an example of the formatting in the eng. playbook.